.github/workflows/dev-image-update.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
+    if: "!contains( github.event.pull_request.labels.*.name, 'version-update')"
```

Comment:
Just realized that this won't work. In our bumper action, we commit directly to develop without a PR, so it wouldn't have a label.